lib/src/DynamicProto.ts

```diff
@@ -152,6 +144,31 @@ function _getObjProto(target:any) {
     return null;
 }
 
+/**
+ * Helper to get the properties of an object, including none enumerable ones as functions on a prototype in ES6
+ * are not enumerable.
+ * @param target 
+ */
+function _forEachProp(target: any, func: (name: string) => void) {
```

Comment:
Needed to add a new lookup function as prototype functions in ES6 are not enumerable.